elevate-project/controllers/project.js

```diff
@@ -1,29 +1,97 @@
+/**
+ * name : controllers/project.js
+ * author : Adithya Dinesh
+ * Date : 22-Aug-2024
+ * Description : Orchestration controller for project
+ */
+
 const routeConfigs = require('../constants/routes')
 const requesters = require('../utils/requester')
-const requestParser = require('../utils/requestParser')
-
-// const createUser = async (req, res, responses) => {
-// 	const selectedConfig = routeConfigs.routes.find((obj) => obj.sourceRoute === req.sourceRoute)
-// 	return await requesters.post(req.baseUrl, selectedConfig.targetRoute.path, req.body)
-// }
-// const updateUser = async (req, res, responses) => {
-// 	const selectedConfig = routeConfigs.routes.find((obj) => obj.sourceRoute === req.sourceRoute)
-
-// 	//const filteredRequestBody = requestParser.transformUpdateUserBody(req.body)
-// 	return await requesters.patch(req.baseUrl, selectedConfig.targetRoute.path, req.body, req.headers)
-// }
-
-// const entityTypeRead = async (req, res, responses) => {
-// 	const selectedConfig = routeConfigs.routes.find((obj) => obj.sourceRoute === req.sourceRoute)
-// 	return await requesters.post(req.baseUrl, selectedConfig.targetRoute.path, req.body, {
-// 		'X-auth-token': req.headers['x-auth-token'],
-// 	})
-// }
-
-const userController = {
-	// createUser,
-	// updateUser,
-	// entityTypeRead,
+const common = require('../constants/common')
+/**
+ * Fetch project templates from projects service.
+ * @name fetchProjectTemplates
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} responses  
+ * @returns {JSON} - List of project templates
+ */
+/**
+ * req.body = {
+ * 		organization_id : 1,
```

Comment:
@priyanka-TL please remove unwanted comments

Reply:
@rakeshSgr @adithyadinesh0412  added this as per Akash comment